Framework/DataHandling/src/LoadEventNexus.cpp

```diff
@@ -9,6 +9,8 @@
 #include "MantidAPI/Sample.h"
```

Comment:
This sounds a little basic, but I would check to see if `LoadEventNexus` expects/gurantees the Instrument to be present (otherwise you'll have null/empty `ComponentInfo`, `DetectorInfo` etc). Sounds stupid, but I did come across this exact problem assuming that `LoadNexusProcessed` required the the instrument information to be present in the file.